cfme/configure/configuration/__init__.py

```diff
@@ -423,8 +428,10 @@ class ServerLogDepot(Pretty):
 
     """
     elements = Region(
-        locators={},
-        infoblock_type="form"
+        locators={
+            "last_message": InfoBlock.create("Basic Info", "Last Message"),
+            "last_log_collection": InfoBlock.create("Basic Info", "Last Log Collection"),
+        },
```

Comment:
I'm just not happy about having to define each field in the locators. Could just be me. ```infoblock.text("Basic Info", "Last Message")``` seems to me to be nicer than first having to define the Basic Info / Last Message infoblock locator to use it